### DISCUSSION TITLE
Option to compress or break up large responses

### DISCUSSION THREAD

**Discussion by mdwelsh** [deepgram/community#743] [Feature Requests]
I'm using pre-recorded transcription on large (several hours long) audio files, however, the responses can be quite large -- often exceeding 5-10MB. When using callbacks, the size of the request body can exceed the typical buffer limits in various frameworks (and such large responses are problematic for memory consumption reasons).

Ideally, there would be an option to have the transcript JSON either compressed or sent back in chunks to the callback.

I might be holding it wrong, but `Accept-Encoding: gzip` does not seem to work.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by nikolawhallon** [reply]
I like the idea of streaming back chunks, e.g. server-side events, would something like that solve these issues? (We don't support this right now though). A quick hack would be to break the audio up client-side.
